picard/ui/options/plugins.py

```diff
@@ -230,46 +233,71 @@ def plugin_updated(self, plugin_name):
                 self.change_details()
                 break
 
+    def uninstall_plugin(self):
+            plugin = self.items[self.ui.plugins.selectedItems()[COLUMN_NAME]]
+            buttonReply = QtWidgets.QMessageBox.question(self, _("Confirmation"),
+                    _("Do you really want to uninstall the plugin %s ?") % plugin.name,
```

Comment:
https://github.com/metabrainz/picard/pull/809#discussion_r163402351

Reply:
sorry about that, missed it unfortunately,